cargo-dist/src/lib.rs

```diff
@@ -1462,6 +1615,16 @@ fn compute_package_info(
     workspace_root: &Utf8Path,
     package: &PackageMetadata,
 ) -> Result<PackageInfo> {
+    // Is there a better way to get the path to sniff?
+    // Should we spider more than just package_root and workspace_root?
+    // Should we more carefully prevent grabbing LICENSES from both dirs?
+    // Should we not spider the workspace root for README since Cargo has a proper field for this?
```

Comment:
i would say we should probably use what we can from cargo- 

cargo also has the license information... we could check if it matches? unclear if that's valuable or not-

Reply:
we do use what we can but license-file is one value and most repos have two (license-file is an escape-hatch for "my license is something weird and custom SPDX has no idea about")